src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -1,4 +1,4 @@
-float calcLightAttenuation( float lightDistance, float cutoffDistance, float decayExponent ) {
+float calcLightAttenuation( const in float lightDistance, const in float cutoffDistance, const in float decayExponent ) {
```

Comment:
no need to have "const in" here really, it just leads to more verbose code and I've never seen it make a performance difference.